addon/components/labs-ui/site-header.js

```diff
@@ -1,19 +1,17 @@
-import Component from '@ember/component';
-import layout from '../../templates/components/labs-ui/site-header';
+import Component from '@glimmer/component';
 
-export default Component.extend({
-  tagName: 'header',
-  classNames: ['site-header'],
+export default class SiteHeaderComponent extends Component {
```

Comment:
is this component tagless? is there a wrapping `div` tag?

Reply:
The <header>  tag got moved to the handlebars already!

Reply:
sweet! is that by default?

Reply:
Yeah! This is one freakin' rad thing about Glimmer components. No more divitis! 

But to clarify, the `<header>` tag didn't get moved to the handlebars. It was mistakenly already there — we had `<header><header>...</header></header>` in our markup, which was noted in #59, and will be resolved here.